sklearn/notebooks/Online Prediction with scikit learn.ipynb

```diff
@@ -0,0 +1,698 @@
+{
```

Comment:
@nnegrey : Could you add a link to this notebook from the top-level README?

Reply:
Added